src/main/java/cc/hyperium/integrations/BetterChatFilter/BetterChatFilter.java

```diff
@@ -0,0 +1,84 @@
+/*
+ *     Copyright (C) 2018  Hyperium <https://hyperium.cc/>
+ *
+ *     This program is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU Lesser General Public License as published
+ *     by the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ *
+ *     This program is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU Lesser General Public License for more details.
+ *
+ *     You should have received a copy of the GNU Lesser General Public License
+ *     along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+// Created by RDIL in November 2018.  
+// Created in hope to limit toxicity.  
+
+package cc.hyperium.integrations.BetterChatFilter;
```

Comment:
All package names should be lowercase. This applies to the folders they reside in as well.